test/unit/metrics/test_group_metric_result.py

```diff
@@ -38,7 +39,7 @@ def test_complex_equality(self):
         assert not(a != b)
         assert not(b != a)
 
-    def test_complex_inequality(self):
+    def dont_test_complex_inequality(self):
```

Comment:
Why are these tests being disabled?

Reply:
A dictionary of numpy arrays throws an exception when called with `==` or `!=` because there are different ways of comparing arrays. These were all tests for `__eq__` and `__ne__` methods that `group_metric_result` was implementing, so I'm not sure that we still need them.

Reply:
Okay to delete these tests?

Reply:
If they're no longer relevant, then delete them. If tests have to be disabled because of a bug, use `pytest.skip`.

Reply:
you can also mark them as `xfail` to _allow them to fail_.

Reply:
i'm removing the test file since we have replaced `group_metric_result` by `Bunch`, so it does not need testing.